src/live-connect.js

```diff
@@ -115,11 +135,30 @@ export function LiveConnect (liveConnectConfig, externalStorageHandler) {
       peopleVerifiedId: postManagedState.data.peopleVerifiedId,
       ready: true,
       resolve: resolver.resolve,
-      resolutionCallUrl: resolver.getUrl
+      resolutionCallUrl: resolver.getUrl,
+      config: liveConnectConfig
     }
   } catch (x) {
     console.error(x)
     emitter.error('LCConstruction', 'Failed to build LC', x)
-    return window.liQ
   }
 }
+
+/**
+ * @param {LiveConnectConfiguration} liveConnectConfig
+ * @param {StorageHandler} externalStorageHandler
+ * @returns {LiveConnect}
+ * @constructor
+ */
+export function LiveConnect (liveConnectConfig, externalStorageHandler) {
+  console.log('Initializing LiveConnect')
+  try {
+    window && (window.liQ = window.liQ || [])
```

Comment:
if the `liQ` is an array, should we push the messages into the initialized LiveConnect as we do in Builder https://github.com/LiveIntent/LiveConnectBuilder/blob/master/src/index.js#L75-L82?